Month1/Week2/Section2/Month1-Week2-Section2/Spotify/PlayList.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "Song.h"
+
+class FPlayList {
+
+private:
+	std::string PlayListTitle;
+	const static int MaxSongCount = 30;
+	int SongCount = 0;
+	FSong Songs[MaxSongCount];
+	
+public:
+	FPlayList();
+	FPlayList(std::string NewPlayListTitle);
+	std::string GetListTitle();
+	void SetListTitle(std::string NewPlayListTitle);
+	bool bAddSong(FSong FSong);
+	FSong GetSong(int SongIndex);
+	void  PlayListDisplay();
```

Comment:
Remember that functions should start with a verb since they should describe what they do. This name would be just fine if it was written the other way around. `DisplayPlaylist` is a lot clearer than `PlaylistDisplay`